packages/plugin-block-charts/src/client/chart/antd/GroupedTable.ts

```
@@ -128,3 +104,93 @@ export class GroupedTable extends AntdChart {
+  };
+
+  return recursiveBuild(data, 0);
+};
+
+const countDataSource = (dataSource, measures, transform) => {
+  measures?.forEach((dataValue) => {
+    const countData = (data) => {
+      data.forEach((value) => {
+        if (!value.children?.length) return;
+        const dataKey = dataValue.field.join('.');
+        if (isNaN(Number(value[dataKey]))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)